modules/acquia_cms_common/src/EventSubscriber/ConfigEventsSubscriber.php

```diff
@@ -64,134 +64,136 @@ public static function getSubscribedEvents() {
    * @throws \Drupal\Core\Entity\EntityStorageException
    */
   public function configSave(ConfigCrudEvent $event) {
-    $moduleInstallTriggered = $this->acmsUtilityService->getModulePreinstallTriggered();
-    if (InstallerKernel::installationAttempted() || $moduleInstallTriggered) {
-      $config = $event->getConfig();
-      // During site install or module install Update views display_options's
-      // style type as cohesion_layout and option's views_template &
-      // master_template if acquia_cms_site_studio module is present
-      // otherwise use default one.
-      if ($this->moduleHandler->moduleExists('acquia_cms_site_studio')) {
-        switch ($config->getName()) {
-          case 'views.view.event_cards':
-            _acquia_cms_common_update_view_display_options_style('event_cards', 'past_events_block', 'view_tpl_event_cards_slider');
-            _acquia_cms_common_update_view_display_options_style('event_cards', 'upcoming_events_block', 'view_tpl_event_cards_slider');
-            break;
-
-          case 'views.view.article_cards':
-            _acquia_cms_common_update_view_display_options_style('article_cards', 'default', 'view_tpl_article_cards_slider');
-            break;
-
-          case 'field.field.node.page.body':
-            _acquia_cms_common_update_page_configurations('field.field.node.page.body', [
-              'label' => 'Search Description',
-              'description' => 'A short description or teaser which will be displayed in search results.',
-            ]);
-            break;
-
-          case 'core.entity_form_display.node.page.default':
-            _acquia_cms_common_update_page_configurations('core.entity_form_display.node.page.default', [
-              'content.field_layout_canvas' => [
-                'type' => 'cohesion_layout_builder_widget',
-                'weight' => 2,
-                'settings' => [],
-                'third_party_settings' => [],
-                'region' => 'content',
-              ],
-            ]);
-            break;
-
-          case 'core.entity_view_display.node.page.default':
-            _acquia_cms_common_update_page_configurations('core.entity_view_display.node.page.default', [
-              'content.field_layout_canvas' => [
-                'type' => 'cohesion_entity_reference_revisions_entity_view',
-                'weight' => 2,
-                'label' => 'hidden',
-                'settings' => [
-                  'view_mode' => 'default',
-                  'link'  => FALSE,
+    if (!\Drupal::isConfigSyncing()) {
```

Comment:
@apathak18 Can you please confirm, if this is working as expected when you run `drush cim` ?

Reply:
Reverted this one